feature-group-definitions/messageerror.yml

```
@@ -0,0 +1,22 @@
+name: messageerror
+description: The `messageerror` event fires on a target, such as a window or worker, when an incoming message cannot be deserialized. This event can fire for many types of messages, such as broadcast channels or cross-document messaging.
```

@ddbeck what do you think of this further edit?

```suggestion
description: The `messageerror` event fires on a target, such as a window or worker, when an incoming message cannot be deserialized. This event can fire for many types of messages, such as cross-document messages or broadcast channel messages.
```

Trying to avoid the reading that a broadcast channel is itself a kind of message.